open-api/rest-catalog-open-api.py

```diff
@@ -905,6 +973,28 @@ class CreateTableRequest(BaseModel):
     properties: Optional[Dict[str, str]] = None
 
 
+class PlanContext(BaseModel):
+    select: List[str] = Field(..., description='A list of the selected columns')
+    project: Optional[List[str]] = Field(None, description='A list of the projections')
+    filter: Optional[Expression] = None
+    options: Dict[str, str]
+    snapshot_id: Optional[int] = Field(None, alias='snapshot-id')
+    from_snapshot_id: Optional[int] = Field(None, alias='from-snapshot-id')
+    to_snapshot_id: Optional[int] = Field(None, alias='to-snapshot-id')
+    timestamp_ms: Optional[int] = Field(None, alias='timestamp-ms')
```

Comment:
alias `timestamp-ms` -> `as-of-time`?

Reply:
yea let me fix that to be in line with `IncrementalDataTableScan`